Library/src/Multiplayer/MultiplayerConnection.cpp

```diff
@@ -442,14 +442,29 @@ void MultiplayerConnection::StartEventMessageListening()
 		}
 		else if (EventType == "SequenceChanged")
 		{
-			if (!SequenceChangedCallback)
+			SequenceChangedEventDeserialiser SequenceDeserialiser;
+			SequenceDeserialiser.Parse(EventValues);
+
+			if (SequenceChangedCallback)
```

Comment:
Created another event here, as we dont want to overwrite the SequenceChanged event, as this could be being used by clients